build/charts/theia/templates/clickhouse/zookeeper/statefulset.yaml

```diff
@@ -0,0 +1,182 @@
+{{- if and (.Values.clickhouse.cluster.enable) (eq (len .Values.clickhouse.cluster.zookeeperHosts) 0) }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: zookeeper
+  labels:
+    app: zookeeper
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: zookeeper
+  serviceName: zookeepers
+  replicas: 3
+  updateStrategy:
+    type: RollingUpdate
+  podManagementPolicy: Parallel
+  template:
+    metadata:
+      labels:
+        app: zookeeper
+        what: node
+      annotations:
+        prometheus.io/port: '7000'
+        prometheus.io/scrape: 'true'
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: "app"
+                    operator: In
+                    values:
+                      - zookeeper
+              topologyKey: "kubernetes.io/hostname"
+      containers:
+        - name: kubernetes-zookeeper
+          imagePullPolicy: IfNotPresent
+          image: "docker.io/zookeeper:3.8.0"
```

Comment:
Shall we have an image hosted on our own dockerhub for the usual reasons?

Reply:
Thanks @salv-orlando for pointing out this. I added the uploading workflow. Could you help to update the theia dockerhub token so that it can have the permission to upload the `theia-zookeeper` image? Thanks a lot!